arangograph/projects.md

```diff
@@ -115,9 +121,14 @@ certificate generation, both the default and high port serve the same
 self-signed certificate.
 {% endhint %}
 
-When a private endpoint is being used, you can specify alternate DNS names,
-which are added to the self-signed certificate only as Subject Alternative
-Name (SAN).
+When using private endpoints, notebooks, and metrics, you can specify 
```

Comment:
can you add a note in the PE section, so a customer can find this info more easily?

Reply:
done